<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanificacionWindow</class>
 <widget class="QMainWindow" name="PlanificacionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlanificacionWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>571</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Flight plan         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="label_mapa">
         <property name="text">
          <string>flight_plan_pruebaQT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_abrir_mapa">
         <property name="text">
          <string>Explorador</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Controlador      </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="label_controlador">
         <property name="text">
          <string>rover2f405WSE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_abrir_controlador">
         <property name="text">
          <string>Explorador</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Archivo conf     </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="label_conf">
         <property name="text">
          <string>conf</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_abrir_conf">
         <property name="text">
          <string>Explorador</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Aircraft               </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="label_aircraft">
         <property name="text">
          <string>Rover_Avelino</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Puntos de paso</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="label_Puntos_paso">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="button_editor">
       <property name="text">
        <string>Abrir flight plan editor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_sectores">
       <property name="text">
        <string>Selecci√≥n de sectores</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_optimizacion">
       <property name="text">
        <string>Optimizador</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_move_wp">
       <property name="text">
        <string>Cargar flight plan</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>571</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="button_estrategia">
       <property name="text">
        <string>Estrategia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_clear">
       <property name="text">
        <string>Limpiar waypoints</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_datos">
       <property name="text">
        <string>Guardar datos</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
